Pre-/Post-Events and Trains - final comment period (#84)
RustFestEU/blog.rustfest.eu: _posts/2019-08-06-pre-post-events-sustainable-trains.md

---
layout: post
title: "Pre-/Post-Events: sustainable train travels from and to RustFest Barcelona"
authors:
  - dns2utf8

> socialImageSrc: "/assets/posts/barcelona/cfp-opens.png"

Reviewer: Needs updated social image.

Author: Yes, I was thinking the map

Author: ![grafik](https://user-images.githubusercontent.com/739070/62707014-28cf5500-b9e0-11e9-92dc-39ee632a2ec6.png)

---

## The big plan 🚆

> What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?

Reviewer: I'd be a bit more wordy here and say what you want to achieve.

```suggestion
What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?
We want to make that happen and are organizing a community travel from the north of Europe all the way down to Barcelona, with stops in between.
We want to encourage more people to join us or plan similar travel events.
```

something like that.

So what do we need for that?

1. Is there a path to go by train?
2. How far can you go in one day?
3. Which stops are interesting?

### Tickets or Interrail

Usually people buy tickets from A to B, sometimes by locking in on a single connection.

> However, a travel of this size is different.

Reviewer: ```suggestion
However, a travel of this length is different.
```

Reviewer: Actually what about slightly rephrasing this:

```suggestion
However, a travel of this length has some challenges.
```

When traveling huge distances over many countries sometimes delays happen or a train might break down.
With a regular ticket you have to find staff and organize a reroute on the spot.

This is where Interrail saves the day.
With the Interrail global pass you buy travel days over a period of time.
So you can simply catch the next train and continue your journey.

> Eg. buy 7 travel days within one months.

Reviewer: This is an example sentence as a standalone one.

I'd merge it with the next sentence:

```suggestion
If you buy 7 travel days within one months, you have the flexibility to ride any train outside of your home country on 7 days.
```

Author: Yes, this sounds better

> If you buy 7 travel days within one months, you have the flexibility to ride any train outside of your home country on 7 days.

Reviewer 2: month

Author: fixed

> So you have the flexibility to ride any train outside of your home country on 7 days.

Reviewer 2: The same thing has been said twice here, so line 32 can be deleted :)
Also, I would write "take any train" instead of "ride"

Author: This is the very explicit way, because most people not familiar with interrail for some reason don't understand it at first.

Reviewer 2: Ok, makes sense :)

In addition, the ticket is also valid for travel in your country of residence during one outbound journey and one inbound journey that occur during travel days at any point in your trip.

Most high-speed and night trains require a [reservation at an additional cost](https://www.interrail.eu/en/plan-your-trip/reservations).

For most European citizens [the Interrail pass (currently 10% off)](https://www.interrail.eu/en/autumn-sale/global-pass) is probably the most flexible and affordable ticket.
For [Non-EWR citizens](https://www.interrail.eu/en/help/ordering-pass/do-i-need-interrail-or-eurail-pass) there is [Eurail](https://www.eurail.com/en/eurail-passes/global-pass).

### Tools

Inside Spain, Portugal, France:

> * [trenes](https://train.trenes.com/) <- Don't forget to uncheck the "booking.com" button

Reviewer: ```suggestion
* [Trenes](https://train.trenes.com/) (Don't forget to uncheck "Also search for accomodation")
```

> Inside Spain, Portugal, France:

Reviewer 2: "Within" instead of "inside"

> For northern Europa:

Reviewer 2: Northern Europe

Author: thanks

* Austria, Germany, Czech Republic, Slovakia, Hungary, Italy, Switzerland, Serbia -> [ÖBB](https://www.oebb.at/)
* Sweden, Denmark, Netherlands, Germany, Switzerland -> [Deutsche Bahn](https://bahn.de)
* Switzerland, France, Spain -> [Schweizerische Bundesbahnen](https://sbb.ch)

## The travel to Barcelona

Now let us look at some examples:

### Within Spain

From Madrid to Barcelona there are two kinds of trains traveling.

* A cheaper train that takes 9.5 hours.

> * With the high speed you can reach Barcelona in 2.5 hours.

Reviewer 2: high speed "train" (word missing)

Author: thanks :)

* From Sevilla Santa Justa you travel directly to Barcelona within 5.5 hours.

### From France or Portugal

From most bigger cities of France you can reach Barcelona within 7 to 9 hours.

* From Paris, there is a direct TGV connection 3 times a day.
* From Rennes, take the TGV via Nimes.
* From Bordeaux, with InterCity and TGV via Narbonne.

* From Lisboa Estacion Oriente there is a train to Madrid Chamartin. Since it is not a high speed train it takes 11 hours and you will probably need to continue on the second day.

Note that there is a [cheaper Interrail country pass](https://www.interrail.eu/en/interrail-passes/one-country-pass/spain-passes) and you probably only need 3 days within 1 month.

### From the North

> These events are timed so that people traveling towards Barcelona can stop by.

Reviewer: It would make sense to mention that this is your route and your taking over planning this.

Reviewer: Also state that this trip is intended to start Wednesday morning and you arrive in Barcelona by Thursday night.

What does that mean, let's go through a possible option:

{:.layout-auto}
| City                            | Day       | Departure | Arrival | Notes                                                                                     |
|---------------------------------|-----------|-----------|---------|-------------------------------------------------------------------------------------------|

> | Gothenburg, SE                  | Wednesday | 08:55     | 12:28   | Swing by the office before and grab a laptop and a coffee to go.                          |

Reviewer: Which office? Is this plan solid already?

Author: Like the office the reader works. Does that make sense?

Author: Removed

| Hamburg, DE                     | Wednesday | 21:02     | 07:54   | Now we can fade out the day and rest.                                                     |
| Basel, CH                       | Thursday  | 08:07     | 09:00   | This is the high speed train, you can stay on the NightJet and arrive a little later.      |
| Zürich HB, CH                   | Thursday  |           | 09:42   | Time to walk a little in Zürich, meet people and maybe hit a shower and a day of hacking. |
| Zürich HB, CH                   | Thursday  | 12:03     | 14:45   | At this point our group might have grown a bit.                                           |
| Genève, CH                      | Thursday  | 14:59     | 18:05   | The fast TGV. 🚅                                                                          |
| Valence TGV Rhône-Alpes Sud, FR | Thursday  | 18:22     | 22:44   | Say goodbye to the Alps.                                                                  |
| Barcelona Sants, ES             | Thursday  |           | 22:44   | 🎉 Hurray, we made it.                                                                       |

### From the East

> Now what if we were able to meet up with people?

Reviewer: This post already has lots of rhetoric questions. I would just state this as a thing:

```suggestion
If your traveling from the east it might be possible to cross routes with folks from the north and join their travel.
```

or similar

> If your traveling from the east it might be possible to cross routes with folks from the north and join their travel.

Reviewer 2: If you are

Reviewer 2: And capitalize "East" and "North" :)

Author: Thank you

> Lucky us there is a direct line from Budapest to Zürich.

Reviewer 2: "connection" instead of "line"

Author: Gerglish ^^

Reviewer 2: Hahahaha :D

Let's look at the connection:

{:.layout-auto}
| City                            | Day       | Departure | Arrival | Notes                                                                                   |
|---------------------------------|-----------|-----------|---------|-----------------------------------------------------------------------------------------|

> | Budapest-Keleti, HU             | Wednesday | 20:40     | 08:20   | Swing by the office before and grab a laptop and a coffee to go.                        |

Reviewer: Again, which office?

| Vienna, AT                      | Wednesday | 23:25     | 08:20   | Same train, has multiple stops in Austria.                                              |
| Zürich HB, CH                   | Thursday  |           | 08:20   | Time to walk a little in Zürich, meet people and maybe hit a shower and day of hacking. |

> Note that you can take a direct train from Warsaw/Warszawa or Prague/Praha to Vienna/Wien.

Reviewer: I think using the english name only would be fine.

> 

Reviewer: How do people coordinate? Is there a contact point? Maybe make it clear that people need to self-organize (and buy their tickets!)

Can you link to the website with something like "We will keep updated information about routes and travel groups on our website"

Author: Our website and the rust users forum?

### From the East

If your are traveling from the East it might be possible to cross routes with folks from the North and join their travel.
Lucky us there is a direct connection from Budapest to Zürich.
Let's look at it:

{:.layout-auto}
| City                | Day       | Departure | Arrival | Notes                                                                                   |
| ------------------- | --------- | --------- | ------- | --------------------------------------------------------------------------------------- |
| Budapest-Keleti, HU | Wednesday | 20:40     | 08:20   | Jump on the NightJet and relax.                                                         |
| Vienna, AT          | Wednesday | 23:25     | 08:20   | Same train, has multiple stops in Austria.                                              |
| Zürich HB, CH       | Thursday  |           | 08:20   | Time to walk a little in Zürich, meet people and maybe hit a shower and day of hacking. |

Note that you can take a direct train from Warsaw or Prague to Vienna.

### From the South

> We will suggest two possible options from Italy. Italy basically has one "train backbone" that goes from South to North so you choice basically is how many stops you want to do. For more options and ideas visit [https://www.interrail.eu](https://www.interrail.eu).

Reviewer 3: ```suggestion
We will suggest two possible options from Italy. Italy basically has one "train backbone" that goes from South to North so you basically choose how many stops you want to do. For more options and ideas visit [https://www.interrail.eu](https://www.interrail.eu).
```

---
layout: post
title: "Pre-/Post-Events: sustainable train travels from and to RustFest Barcelona"
authors:
  - dns2utf8
socialImageSrc: "/assets/posts/barcelona/cfp-opens.png"
---

## The big plan

> What if you could meet with other rusteacens while you travel to and from RustFest Barcelona?

Reviewer: ```suggestion
What if you could meet with other Rustaceans while you travel to and from RustFest Barcelona?
```

> What if you could meet with other rusty people while you travel to and from RustFest Barcelona?

Reviewer 3: ```suggestion
What if you could meet with other rusteacens while you travel to and from RustFest Barcelona?
```

So what do we need for that?

1. Is there a path to go by train?
2. How far can you go in one day?
3. Which stops are interesting?

### Tools

For northern Europa:
* Switzerland, Germany, Netherlands, Denmark, Sweden -> [Deutsche Bahn](https://bahn.de)
* Switzerland, France, Spain -> [Schweizer Bundes Bahn](https://sbb.ch)

## Pre-Events

These events are timed so that people traveling towards Barcelona can stop by.

> What does that mean, let's go trough a possible option:

Reviewer 3: ```suggestion
What does that mean, let's go through a possible option:
```

| City                            | Day       | Departure | Arrival | Notes                                                                                   |
|---------------------------------|-----------|-----------|---------|-----------------------------------------------------------------------------------------|
| Gothenburg, SE                  | Wednesday | 08:55     | 12:28   | Swing by the office before and grab a laptop and a coffee to go.                        |
| Kopenhagen, DK                  | Wednesday | 12:52     | 18:02   | Via Fredericia st.                                                                      |
| Hamburg, DE                     | Wednesday | 18:02     |         | Here we could have a meetup in Hamburg, probably best with food.                        |
| Hamburg, DE                     | Wednesday | 21:02     | 07:54   | Now we can fade out the day and rest.                                                   |
| Basel, CH                       | Thursday  | 08:07     | 09:00   | This is the highspeed train, you can stay on the NightJet and arrive a little later.    |

> | Zürich HB, CH                   | Thursday  |           | 09:42   | Time to walk a little in Zürich, meet people and maybe hit a shower and day of hacking. |

Reviewer 3: ```suggestion
| Zürich HB, CH                   | Thursday  |           | 09:42   | Time to walk a little in Zürich, meet people and maybe hit a shower and a day of hacking. |
```

| Zürich HB, CH                   | Thursday  | 12:03     | 14:45   | At this point our group might have grown a bit.                                         |
| Genève, CH                      | Thursday  | 14:59     | 18:05   | The fast TGV.                                                                           |

> | Valence TGV Rhône-Alpes Sud, FR | Thursday  | 18:22     | 22:44   | Say good by to the alps.                                                                |

Reviewer 3: ```suggestion
| Valence TGV Rhône-Alpes Sud, FR | Thursday  | 18:22     | 22:44   | Say goodbye to the Alps.                                                                |
```
